storage/operator.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: rook-operator
+  namespace: default
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: rook-operator
+    spec:
+      containers:
```

Comment:
The rook-operator talks to the K8s API for resource creation (pods, service, configmaps, secrets, CRDs etc). Therefore it needs to have permission to do all that. If you setup k8s with RBAC, then you will need to provide a serviceaccount that have permissions to do this.

Reply:
For the sake of simplicity and within the context of the Hobby Kube, I recommend using a permissive RBAC configuration. Details can be found here: https://github.com/hobby-kube/guide#role-based-access-control